### ISSUE TITLE
Unable to use LetXPath in Chrome Incognito window

### ISSUE THREAD

**Issue by koushikbiz** [ortoniKC/LetXPath#13]
LetXPath doesn't generate XPath in Chrome Incognito window

**To Reproduce**
1. I enabled LetXPath for Chrome Incognito mode
2. Opened an Incognito window and enabled LetXPath
3. Selected an element
4. Element XPath details are not generated

**Expected behavior**
Element XPath details should be generated in Incognito window

**Desktop (please complete the following information):**
 - OS: MacOS Venture 13.0
 - Browser - Chrome
 - Version - 125.0.6422.113 (Official Build) (arm64)

**Comment by koushikbiz** [comment]
![Uploading Screenshot 2024-06-19 at 10.37.49 PM.png…]()

**Comment by ortoniKC** [comment]
Is the same works in normal mode?

**Comment by koushikbiz** [comment]
Yes, LetXPath works in normal mode for me. Please let me know if there is any limitation in running it from Incognito window.

**Comment by ortoniKC** [comment]
Seems like an issue with permission, I'll check and provide an update.

**Comment by koushikbiz** [comment]
Thanks. Looking forward to your update.

**Comment by Shilo-design** [comment]
Hi, I am struggling to make it work on my laptop. Do you think I am doing anything wrong. I will take any kind of criticism and help you can offer.

**Comment by ortoniKC** [comment]
I haven't started to work on this, I'll do it ASAP

**Comment by Shilo-design** [comment]
Thank you
